pkg/collector/corechecks/ebpf/probe/tcp_queue_length.go

```diff
@@ -137,6 +145,12 @@ func (t *TCPQueueLengthTracer) Get() TCPQueueLengthStats {
 
 func (t *TCPQueueLengthTracer) GetAndFlush() TCPQueueLengthStats {
 	result := t.Get()
-	t.statsMap.DeleteAll()
+	var statsKey C.struct_stats_key
+	it := t.statsMap.Iterate()
+	for it.Next(unsafe.Pointer(&statsKey), nil) {
+		if err := t.statsMap.Delete(statsKey); err != nil {
```

Comment:
Should this be `unsafe.Pointer(&statsKey)`?

Reply:
Indeed